hyper/hyper/restore_test.go

```diff
@@ -0,0 +1,57 @@
+package hyper_test
+
+import (
+	"testing"
+
+	"github.com/isard-vdi/isard/hyper/hyper"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRestore(t *testing.T) {
+	require := require.New(t)
+	assert := assert.New(t)
+
+	cases := map[string]struct {
+		PrepareDesktop func(h *hyper.Hyper) string
+		ExpectedErr    string
+	}{
+		"restore the desktop correctly": {
```

Comment:
There should be more checks here. Should the desktop be running after being restored?

Reply:
I don't get this. It is done on test run:
`			err = h.Restore(path)

			if tc.ExpectedErr == "" {
				assert.NoError(err)
			} else {
				assert.EqualError(err, tc.ExpectedErr)
			}`

Reply:
I see, fixed